### DISCUSSION TITLE
Is it possible to change the error bag used by livewire validation?

### DISCUSSION THREAD

**Discussion by Rattone** [livewire/livewire#7968] [1. Help]
The scenario is this: I have a page with a standard login form and, in the footer, a livewire component for subscribing to the newsletter.

Both forms have an "email" field. When an error occurs in the "email" field of the standard login form, the error is also shown in the livewire component because they use the same error bag.

I don't want to change the error bag of the various standard forms but I would like to customize the bag of the livewire component.

**Comment by joshhanley** [comment]
@Rattone this is a duplicate of #2585 so going to close this one.
